### ISSUE TITLE
Syntax Error in local_defaults.py.example

### ISSUE THREAD

**Issue by andyprata** [ORAC-CC/orac#81]
https://github.com/ORAC-CC/orac/blob/2180705f9bf8050cddc548d0b0321da10ad9f57f/tools/pyorac/local_defaults.py.example#L156

Should be: `"summary": "n/a"`

**Comment by adamcpovey** [comment]
Yeah, those should all be fixed. Go ahead if you've already done it.

I've never actually tried to use the file and am slightly tempted to replace it with the JASMIN one to remove a step for new users.
